docs/en/FAQ/Protoc-Plugin-Fails-When-Build.md

```diff
@@ -0,0 +1,9 @@
+**Problem**：
+- When the Collector starts, maven build the protoc-plugins to produce the error “No such file or directory”.
```

Comment:
still says collector starts, and you should put your error logs here in markdown format.  Same in CN document.

Reply:
ok